src/installer.py

```diff
@@ -111,48 +105,28 @@ def uninstall(self, password):
         
         #Starting bash and getting root privileges
         print("Starting bash and getting root privileges")
-        pyotherside.send('whatState',"=> Starting uninstaller")
-        time.sleep(1.5)
+        pyotherside.send('state', 'starting', False)
         child = pexpect.spawn('bash')
+        child.logfile_read = sys.stdout.buffer
         child.expect(r'\$')
         child.sendline('sudo -s')
         if password != '':
             child.expect('[p/P]ass.*')
             child.sendline(str(password))
         child.expect('root.*')
 
-        #remounting filesystem to rw
-        print("remounting filesystem to rw")
-        pyotherside.send('whatState',"=> remounting filesystem as readwrite")
-        time.sleep(1.5)
-        child.sendline('mount -o remount,rw /')
-        child.expect('root.*')
-
         #Stop Waydroid Container
         print("stopping waydroid container")
-        pyotherside.send('whatState',"=> stopping waydroid container")
+        pyotherside.send('state', 'container', False)
         child.sendline("service waydroid-container stop")
         child.expect("root.*", timeout=180)
-     
-        #Purge Waydroid
-        print("purging Waydroid")
-        pyotherside.send('whatState',"=> uninstalling Waydroid")
-        child.sendline("apt purge --autoremove waydroid -y")
-        child.expect('root.*', timeout=480)
 
         #do cleanup
         print("cleaning")
-        pyotherside.send('whatState',"=> cleaning up")
-        time.sleep(1.5)
+        pyotherside.send('state', 'cleanup', False)
         child.sendline("rm -rf /var/lib/waydroid")
```

Comment:
Removes the files fine but emits this in `journalctl --user -f`:
```
Jan 30 17:32:57 ubuntu-phablet aa-exec[34835]: root@ubuntu-phablet:/home/phablet# rm -rf /var/lib/waydroid
Jan 30 17:32:57 ubuntu-phablet aa-exec[34835]: rm: cannot remove '/var/lib/waydroid': Read-only file system
```
Should just use `/var/lib/waydroid/*` assuming it's fine with globs.

To fully implement #32 I  suppose this should also `rm -f /home/phablet/.local/share/applications/Waydroid.desktop` (it won't work anyway if images are gone) with an option also remove app data which would additionally `rm -rf /home/phablet/.local/share/{waydroid,applications/waydroid.*.desktop}`

Reply:
Seems like it's fine with globs, but just basic ones - brackets don't work at all

Reply:
> Seems like it's fine with globs, but just basic ones - brackets don't work at all

Use `glob.glob` then for each result call rm?

Reply:
What for? It just resulted in two calls to bash instead of one, no point in complicating it further just to get one.

Reply:
> What for? It just resulted in two calls to bash instead of one, no point in complicating it further just to get one.

Doesn't rm supports multiple files, join the result of glob to a string delimited by spaces, and have just one subprocess called instead of 100000

Reply:
It's just one subprocess now, I don't think I follow what you mean - `*` globs work fine.